packages/ods-react/src/components/combobox/src/context/useCombobox.tsx

```diff
@@ -0,0 +1,59 @@
+import { type ReactNode, createContext, useContext } from 'react';
+
+export type ComboboxOptionItem = {
+  label: string;
+  value: string;
+  disabled?: boolean;
+  customRendererData?: Record<string, unknown>;
+  isNew?: boolean;
+  isInGroup?: boolean;
```

Comment:
those two should be handle separately or marked as private to not appear on final doc